add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The code directly includes SQL queries within the PHP script, which can lead to SQL injection vulnerabilities.

Recommendations: Use prepared statements to protect against SQL injection. Consider using PDO or MySQLi with prepared statements for safer database interactions.

Example:
```php
$stmt = $connection->prepare("INSERT INTO books (title, author, year) VALUES (?, ?, ?)");
$stmt->bind_param("ssi", $title, $author, $year);
$stmt->execute();
```